eng/helix/content/installPlaywrightReqs.ps1

```diff
@@ -0,0 +1,6 @@
+Write-Host "Enable-WindowsOptionalFeature -Online -FeatureName ServerMediaFoundation  (For Playwright)"
```

Comment:
Will file an issue to ask helix infrastructure to enable this feature on all the windows images we use, but we will need this for a while